### DISCUSSION TITLE
What signature method is used in zksync lite?

### DISCUSSION THREAD

**Discussion by agentjacker** [zkSync-Community-Hub/zksync-developers#554] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

When prompting a transaction for zksync lite https://lite.zksync.io for example sending a transfer from one wallet to another. aftere the first signature, the second one is requested and shows the params of the request like amount, gas. what method is used in this request. is it eth_sign, personal sign or eth_signTypedData_v4 ?

**Comment by bxpana** [comment]
Hey thanks for the post. Will look into that for you. 

Out of curiosity, is there a reason you're using zkSync Lite and not zkSync Era? zkSync Era is general purpose where zkSync Lite is restricted to payments and NFTs.

**Reply by agentjacker** [reply]
Would appreciate you getting back thanks!.
just testing areound and learning! i use zksync era too

**Reply by agentjacker** [reply]
hey just pinging again any updates or info on this?

**Reply by bxpana** [reply, marked as answer]
Thanks for your patience on this, the method used in this request is `eth_signTypedData` https://docs.constellationnetwork.io/stargazer/apireference/ethereumrpcapi/eth_signtypeddata/

**Reply by agentjacker** [reply]
Is this the same method used in checkout.zksync.io?

**Reply by bxpana** [reply]
Thanks for your patience. From what I have gathered, yes it should be the same signature for checkout.zksync.io
